crates/modelardb_manager/src/cluster.rs

```diff
@@ -188,17 +171,46 @@ impl Cluster {
             body: table_name.to_owned().into(),
         };
 
-        let mut drop_table_futures: FuturesUnordered<_> = self
+        self.execute_cluster_action(action, key).await
+    }
+
+    /// For each node in the cluster, use the `TruncateTable` action to truncate the table given by
+    /// `table_name`. If the table was successfully truncated for each node, return [`Ok`], otherwise
+    /// return [`ClusterError`](ModelarDbError::ClusterError).
+    pub async fn truncate_tables(
```

Comment:
wh is it called truncate_table*s*? Isn't only one (distributed) table truncated?

Reply:
Fixed.